tools/depends/native/TexturePacker/src/TexturePacker.cpp

```diff
@@ -286,7 +286,7 @@ void Usage()
   puts("  -help            Show this screen.");
   puts("  -input <dir>     Input directory. Default: current dir");
   puts("  -output <dir>    Output directory/filename. Default: Textures.xpr");
-  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: on");
+  puts("  -dupecheck       Enable duplicate file detection. Reduces output file size. Default: off");
```

Comment:
Nope, the `dupecheck` variable is `false` by default and only set to `true` if one of the arguments matches `-dupecheck` (yes the syntax with `!strcmp()` looks confusing).